index.bs

```diff
@@ -273,7 +273,7 @@ The {{MLModelBuilder}} interface defines a set of operations as identified by th
 typedef record<DOMString, MLOperand> MLNamedOperands;
 
 dictionary MLBuffer {
-  required (ArrayBufferView or WebGLBuffer or GPUBuffer) resource;
+  required (ArrayBuffer or WebGLBuffer or GPUBuffer) resource;
```

Comment:
BTW, i think the field name `buffer` seems better. Because it is used in the `MLInput` as the field name is also `resource`. There will be double 'resource' to create a `MLInput`, e.g. `{'input': {resource: {resource: arrayBuffer}}`.